package.json

```diff
@@ -46,7 +46,9 @@
     "luxon": "^3.3.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-redux": "^9.1.0",
```

Comment:
Do we need `react-redux` since we are binding action creators?

Reply:
Yeah, we don't need this.